cloudinit/cmd/devel/hotplug_hook.py

```diff
@@ -216,6 +216,22 @@ def handle_hotplug(hotplug_init: Init, devpath, subsystem, udevaction):
         action=udevaction,
         success_fn=hotplug_init._write_to_cache,
     )
+    start = time.time()
+    elapsed = 0.0
+    if not datasource.hotplug_retry_settings.force_retry:
+        try_hotplug(subsystem, event_handler, datasource)
+        LOG.info("Not forcing retry")
+        return
+    while elapsed < datasource.hotplug_retry_settings.sleep_total:
+        try_hotplug(subsystem, event_handler, datasource)
+        LOG.info(
```

Comment:
Most of the other hotplug related logs are debug. Any particular reason this should be info?